If/then/else is not so different (#28514)
dotnet/docs: docs/fsharp/language-reference/conditional-expressions-if-then-else.md

In the previous syntax, *expression1* runs when the Boolean expression evaluates to `true`; otherwise, *expression2* runs.

> Like other languages, the `if...then...else` construct can be used to conditionally execute code. In F#, however, `if...then...else` is an expression and must also produce a value, being the value of the last expression in the branch that executes. The types of the values produced in each branch must match. If there is no explicit `else` branch, its type is `unit`. Therefore, if the type of the `then` branch is any type other than `unit`, there must be an `else` branch with the same return type. When chaining `if...then...else` expressions together, you can use the keyword `elif` instead of `else if`; they are equivalent.

Reviewer: ```suggestion
Like other languages, the `if...then...else` construct can be used to conditionally execute code. In F#, `if...then...else` is an expression and produces a value by the branch that executes. The types of the expressions in each branch must match.

If there is no explicit `else` branch, the overall type is `unit`, and the type of the `then` branch must also be `unit`. 

When chaining `if...then...else` expressions together, you can use the keyword `elif` instead of `else if`; they are equivalent.
```
